src/client.js

```diff
@@ -69,6 +89,27 @@ export default class Client {
     return this._performSearch(params).then(formatResultsJSON);
   }
 
+  /**
+   * Sends multiple search requests to the Swiftype App Search Api, using the
```

Comment:
It would be good to have that link, however, we don't include a link on any of the other methods, in this client or any other client. So for the sake of convention, I think I'll leave it as is, for now, as long as you think that's OK.